cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/UnmanagedClusterInventory.tsx

```diff
@@ -1,32 +1,72 @@
 // React imports
-import React from 'react';
+import React, { ChangeEvent, MouseEvent, useContext, useEffect, useState } from 'react';
```

Comment:
I think ChangeEvent, MouseEvent, useContext are not used, can remove

Please also check other files that you added or modified for unused imports while you're at it.